### ISSUE TITLE
The localhost of gradio server send a blank page.

### ISSUE THREAD

**Issue by HuangChiEn** [gradio-app/gradio#1929]
As the title describes, I have built a gradio app in the docker of the server. 
This morning i restart the server and run the container, and then active the gradio code with enable the share flag, both of the local, remote server send me a blank page (even without inner-built "loading page symbol" of gradio).

```
...omit ...

if __name__ == "__main__":
    ...omit..
    blk_cntr.launch(share=True, enable_queue=True)
```

The preview page and port forward information in vscode is shown in the following figure.

![image](https://user-images.githubusercontent.com/52521165/182318431-1acc2c57-8093-4ec9-8d1f-dee86e47ad14.png)

![image](https://user-images.githubusercontent.com/52521165/182318479-564348cd-82c9-4278-b75b-e501ea9a9e82.png)

While i have checked the HTML source of the blank page, it does have something:
![image](https://user-images.githubusercontent.com/52521165/182319175-c42ddcf0-bfc3-4e87-957b-d98ca51d53c1.png)
 (PS. i have omit most of the output, but i found those elements in my code is appears in this js script content)

Any suggestion will be appreciated !!

**Comment by HuangChiEn** [comment]
It seems the gradio==3.1.2 (latest version) have some issue to present the page. This problem is fixed while I downgrade to gradio==3.1.0.

![image](https://user-images.githubusercontent.com/52521165/182320715-15bc5128-3a7d-462f-96cb-1a3ae245dba0.png)
( PS. omit the page content, but the loading symbol is shown in above figure

**Comment by abidlabs** [comment]
Hi @HuangChiEn! Thanks for creating the issue.

We made a mistake when releasing 3.1.2 that caused the blank page to occur. We've yanked that release and issued a new release 3.1.3 that should work just fine!

Closing the issue, but let us know if you experience any other issues.

**Comment by HuangChiEn** [comment]
> Hi @HuangChiEn! Thanks for creating the issue.
> 
> We made a mistake when releasing 3.1.2 that caused the blank page to occur. We've yanked that release and issued a new release 3.1.3 that should work just fine!
> 
> Closing the issue, but let us know if you experience any other issues.

yes, it's fine while I update to the patched version 3.1.3, thanks for solving this issue in few hours.
